views/wiz/geometrystep.html

```diff
@@ -133,53 +133,7 @@
-          <select class="custom-select col-2" ng-model="step.ellipse.radius.units" ng-change="geomStep.validate()"
-              ng-disabled="!step.ellipse.radius.column" ng-options="unit for unit in geomStep.units"></select>
-        </div>
-        <div class="form-row form-group">
-          <label class="col-form-label col-2">Semi Major</label>
-          <select class="custom-select col-4 mr-1" name="semiMajor" ng-model="step.ellipse.semiMajor.column" ng-change="geomStep.validate()"
-              ng-required="step.showEllipse && (step.ellipse.semiMinor.column || step.ellipse.orientation.column)"
-              ng-options="column.name as column.name for column in geomStep.columns">
-            <option value="">-- None --</option>
-          </select>
-          <label class="col-form-label col-auto">Units</label>
-          <select class="custom-select col-2" ng-model="step.ellipse.semiMajor.units" ng-change="geomStep.validate()"
-              ng-disabled="!step.ellipse.semiMajor.column"
-              ng-options="unit for unit in geomStep.units"></select>
-          <small class="form-text invalid-feedback d-inline-block col-2" ng-if="ellipseForm.semiMajor.$error.required">Required!</small>
-        </div>
-        <div class="form-row form-group">
-          <label class="col-form-label col-2">Semi Minor</label>
-          <select class="custom-select col-4 mr-1" name="semiMinor" ng-model="step.ellipse.semiMinor.column" ng-change="geomStep.validate()"
-              ng-required="step.showEllipse && (step.ellipse.semiMajor.column || step.ellipse.orientation.column)"
-              ng-options="column.name as column.name for column in geomStep.columns">
-            <option value="">-- None --</option>
-          </select>
-          <label class="col-form-label col-auto">Units</label>
-          <select class="custom-select col-2" ng-model="step.ellipse.semiMinor.units" ng-change="geomStep.validate()"
-              ng-disabled="!step.ellipse.semiMinor.column"
-              ng-options="unit for unit in geomStep.units"></select>
-          <small class="form-text invalid-feedback d-inline-block col-2" ng-if="ellipseForm.semiMinor.$error.required">Required!</small>
-        </div>
-        <div class="form-row">
-          <label class="col-form-label col-2">Orientation</label>
-          <select class="custom-select col-4 mr-1" name="orientation" ng-model="step.ellipse.orientation.column" ng-change="geomStep.validate()"
-              ng-required="step.showEllipse && (step.ellipse.semiMajor.column || step.ellipse.semiMinor.column)"
-              ng-options="column.name as column.name for column in geomStep.columns">
-            <option value="">-- None --</option>
-          </select>
-          <small class="form-text invalid-feedback d-inline-block col-2" ng-if="ellipseForm.orientation.$error.required">Required!</small>
-        </div>
-      </div>
+      <ellipsecolumns layer="config"></ellipsecolumns>
```

Comment:
The `<div class="card-body">` is still needed here to provide some padding around the controls inside the card.